corehq/apps/userreports/sql/adapter.py

```diff
@@ -216,6 +228,24 @@ def doc_exists(self, doc):
             query = session.query(self.get_table()).filter_by(doc_id=doc['_id'])
             return session.query(query.exists()).scalar()
 
+    def get_rows(self, doc_id):
+        """Returns a list of entries from the datasource matching the `doc_id`. Each entry is a dictionary with
+        the column name as the key and value as the value
+        """
+        table = self.get_table()
+        headers = list(table.columns.keys())
+
+        named_entries = []
```

Comment:
nit: this naming seems confusing. I believe you could simply call this rows.

Reply:
c43a584ad43bf8dddffed31948287f2fecd0bf98